src/util/ClientServiceUtility.java

```diff
@@ -0,0 +1,60 @@
+package util;
+
+import entities.BaseEntity;
+import entities.User;
+import java.util.Optional;
+
+public final class ClientServiceUtility {
+  public static BaseEntity updateBaseEntity(BaseEntity baseEntityToUpdate, BaseEntity updated) {
+    baseEntityToUpdate.setUpdatedAt(
+        Optional.ofNullable(updated)
+            .map(BaseEntity::getUpdatedAt)
+            .orElse(baseEntityToUpdate.getUpdatedAt()));
+    baseEntityToUpdate.setCreatedAt(
+        Optional.ofNullable(updated)
+            .map(BaseEntity::getCreatedAt)
+            .orElse(baseEntityToUpdate.getCreatedAt()));
+    baseEntityToUpdate.setCreatedBy(
+        Optional.ofNullable(updated)
+            .map(BaseEntity::getCreatedBy)
+            .orElse(baseEntityToUpdate.getCreatedBy()));
+    baseEntityToUpdate.setUpdatedby(
+        Optional.ofNullable(updated)
+            .map(BaseEntity::getUpdatedby)
+            .orElse(baseEntityToUpdate.getUpdatedby()));
+    baseEntityToUpdate.setStatus(
+        Optional.ofNullable(updated)
+            .map(BaseEntity::getStatus)
+            .orElse(baseEntityToUpdate.getStatus()));
+    return baseEntityToUpdate;
+  }
+
+  public static User updateUser(User userToUpdate, User updated) {
```

Comment:
Move this to a other file `UserUtil` rename to `fetchUser` and rename args `target` and `source`.

Reply:
solved